articles/systems/08-platform-framework.md

```diff
@@ -0,0 +1,49 @@
+# Mixed Reality Toolkit Platform Framework
+
+A major component of the Mixed Reality Toolkit is the Platform Framework that was employed to both accurately determine what platforms have been added to the scope of the project and also to determine when a platform is active/running.
+
+## Platform Selection
+
+![](../images/Configuation/Platforms/PlatformSelector.png)
+
+Wherever configuration can have a variant for platforms, to make the configuration only active when selected platforms are available, the Platform selector is employed.
+
+The list support multiple selections (for cases where a system or profile) can operate for multiple platforms.
+
+The list is dynamic and will update based on which XRTK platforms have been installed and what platforms are supported by the developers Unity installation.
+
+## Platform Identification
+
+To explain how this system basically operates, there is a `Platform` script in the UPM package runtime folder which implements the required interfaces for the platform system, including:
+
+```csharp
+    public override bool IsAvailable;
+```
+
+`IsAvailable` indicates whether the platform is available at runtime, usually checking for a vendor SDK or other API call which signifies the platform is available.
```

Comment:
I'd rather see this documented in the `BasePlatform` or `IMixedRealityPlatform` documentation in the class itself

Reply:
This section is purely a highlight of that area and the advances we have made.  That documentation should already be in the API auto-generated documentation.  We can evolve that further in 0.3